Contacts/cite.mixer

```diff
@@ -43,7 +43,8 @@ href="/Packages/packages.html">package overview</a> page.)
 
 <p>
   If you are using BibTeX, you can use the following BibTeX entry for the
-  current GAP version:
+  current GAP version, making sure you have `\usepackage{url}' (or `\usepackage{hyperref}'
```

Comment:
You need to use `<code>..</code>` for this

Reply:
fixed by https://github.com/gap-system/GapWWW/pull/137/commits/2cad8730e9f743eef5098c4d5a8261abff320a5a